src/utils/FirebaseLib.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactFire from 'reactfire';
```

Comment:
`import ReactFire from 'reactfire';`

This should be `import Firebase from 'firebase';` no? ReactFire is the mixin..